### ISSUE TITLE
Feature request: land labels

### ISSUE THREAD

**Issue by superrache** [shortbread-tiles/shortbread-docs#67]
Thanks for this work, shortbread fulfills its mission of being generalist and minimalist. I don't know if it's the right place for feature request, so let me know if there are some discussion elsewhere.

It would be very useful if the `land` layer had a `name` field (and translations) or if there was an additional `land_labels` point layer. This would allow displaying names for parks, cemeteries, forests, industrial/retail areas, ... which are important landmarks on a map, without overloading tile size too much.
Also, a field `way_area` would allow to display only large areas depending on zoom level.

**Comment by almccon** [comment]
Yes, labels for parks is one of the features I really would like to see!

**Comment by pnorman** [comment]
The land layer is polygons so this would need to be an additional land_labels layer

How's this for a proposal

> ## Properties
> Field name | Type | Description
> -- | -- | --
> kind | string | kind of feature (see *land* layer for a list of values) 
> way_area | float | area in square meter (Mercator Projection)
> name | string | value of OSM name=* tag (with other languages however we decide to in 1.1)

Are there any features in the land layer we wouldn't want to include in a land_labels layer?
